.buildkite/serve.rayci.yml

```diff
@@ -147,6 +147,22 @@ steps:
         --parallelism-per-worker 3
     depends_on: servebuild
 
+  - label: ":ray-serve: serve: HA integration tests"
+    tags:
+      - serve
+      - python
+    instance_type: medium
+    commands:
+      - bazel run //ci/ray_ci:build_in_docker -- docker --python-version 3.8 --platform cpu --canonical-tag ha_integration
+      - docker tag rayproject/ray:ha_integration ray_ci:v1
```

Comment:
where is the `ray_ci:v1` tag is defined? maybe just us the same name overthere and remove this line?

Reply:
it is in the `python/ray/tests/conftest.py`. I can do it as a follow up (and cleanup the old `prepare_docker()` function entirely.